### ISSUE TITLE
[Bug Report][3.3.21] Accessibility: v-autocomplete needs to announce number of potential matches

### ISSUE THREAD

**Issue by kmmerc18** [vuetifyjs/vuetify#18492]
### Environment
**Vuetify Version:** 3.3.21
**Vue Version:** 3.3.4
**Browsers:** Chrome 117.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Go to the example page and use a screen reader (such as VoiceOver for Mac) and begin typing in a v-autocomplete. The screen reader does not announce how many matches appear in the autocomplete list, and there is no way to select/navigate the options without a mouse.

### Expected Behavior
It should be possible using up and down arrows to explore the list, and a screen reader should read out the number of options, as well as each individual option

### Actual Behavior
The screen reader only reads the letters you've typed, and has no indication of a change to the autocomplete list. There is no way to navigate the autocomplete list with a keyboard

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVT8sOgjAQ/JVmL1wU4uNk0MSQ6A+YeAAOFVbSpHSbUojG+O+WIiKnnZ2d3Z1JX9CYIjpqHXYtwg5ii7WW3OIhU4zF3ZJr7aFvClKWC4XmSw2K1lJBbgstjjRjkt9Q7jM4/k9hmu+E+9Q4QRokXIo7GSV4sGBBQpIML6nHJwdF6ekzkqkGxQUfvOnB9Um1UFWQT4cPcTR3NJp39Ny9F/pwcfQLDe8FbMJNuIWhrleQfwDa2lpB)
